### DISCUSSION TITLE
Is this still maintained?

### DISCUSSION THREAD

**Discussion by schooldanlp6** [LMMS/lmms#6573] [Support request] [answered]
Just curious because the latest release was 2020

**Comment by RiedleroD** [comment, marked as answer]
yes, there's been a large reorganisation going on and development in general has been slow because the team is small and - most importantly - volunteers.

The dev builds are recommended right now, because the stable one is so old, though someone else is probably more qualified than me to say this.

iirc there have been talks about a 1.3.0 release, but idk how long it'll take to get there. I think it might take more than another year though, so don't get your hopes up.

**Reply by schooldanlp6** [reply]
Could i Apply as a dev? (and to be fair i might or might not be a good help as i need to get a bit into project)

**Reply by zonkmachine** [reply]
You don't apply to become a dev. Just compile the code and start fixing bugs.
Here are the open issues that have been marked suitable for someone just starting out coding:
https://github.com/LMMS/lmms/issues?q=is%3Aopen+is%3Aissue+label%3A%22good+first+issue%22

Please join our Discord channel and ask away there. You will get more answers there and much faster.

**Comment by zonkmachine** [comment]
@schooldanlp6 Question answered! :)

Please join our Discord channel where the developers and users hang out.
https://discord.com/invite/3sc5su7

**Comment by hakro** [comment]
Dear contributors,

If it makes sense, would it be possible to release a 1.3.0.alpha-2 ?
The alpha-1 is 2 years old, and there seems to be quite a few improvements since then. And this will allow people who are not necessarily tech savvy to give it a test :)

Thank you a lot.
Cheers
